skyportal/models.py

```diff
@@ -112,24 +116,7 @@ class Source(Base):
 
     score = sa.Column(sa.Float, nullable=True)
 
-    ## pan-starrs
-    sgmag1 = sa.Column(sa.Float, nullable=True)
```

Comment:
@profjsb these are the columns we're moving to the single `other_metadata` JSONB column. Any thoughts/suggestions?